scarb/src/ops/cache.rs

```diff
@@ -0,0 +1,13 @@
+use anyhow::{Context, Result};
+
+use crate::core::Config;
+use crate::internal::fsx;
+
+#[tracing::instrument(skip_all, level = "debug")]
+pub fn cache_clean(config: &Config) -> Result<()> {
+    let path = config.dirs().cache_dir.path_unchecked();
+    if path.exists() {
+        fsx::remove_dir_all(path).context("failed to clean cache")?;
+    }
```

Comment:
I smell an issue here. If there are multiple instances of Scarb running, they are synchronizing their cache access, in order to avoid conflicts and other bad stuff. This operation should highly probably acquire all locks possible, because I fear we might end up getting corrupted state in this scenario....

It's actually a probable thing: you might want to clear the cache, having and IDE with LSP running in background.